src/helpers/utils.js

```diff
@@ -470,11 +471,20 @@ export function validateLayout(layout: Layout, contextName: string): void {
         throw new Error('VueGridLayout: ' + contextName + '[' + i + '].' + subProps[j] + ' must be a number!');
       }
     }
-    if (item.i && typeof item.i !== 'string') {
-      // number is also ok, so comment the error
-        // TODO confirm if commenting the line below doesn't cause unexpected problems
-      // throw new Error('VueGridLayout: ' + contextName + '[' + i + '].i must be a string!');
+
+    if (item.i === undefined || item.i === null) {
+      throw new Error('VueGridLayout: ' + contextName + '[' + i + '].i cannot be null!');
+    }
+
+    if (typeof item.i !== 'number' || typeof item.i !== 'string') {
```

Comment:
Shouldn't this be:

`
    if (typeof item.i !== 'number' && typeof item.i !== 'string') {
`

Reply:
Agree, thanks for the comments, I will modify it based on this.